scripts/ci/chainerx/jenkins/run.sh

```diff
@@ -75,6 +75,11 @@ case "${CHAINERX_JENKINS_TEST_TYPE}" in
         CHAINERX_NVCC_GENERATE_CODE=arch=compute_50,code=sm_50 MAKEFLAGS=-j16 run_step python_build
         run_step python_test_chainerx
         ;;
+    'chainerx-py3-cc61')
+        run_step setup_conda_environment
+        CHAINERX_NVCC_GENERATE_CODE=arch=compute_61,code=sm_61 MAKEFLAGS=-j16 run_step python_build
```

Comment:
Please write a comment about why 61.

Reply:
Considering the test for TensorCore, should compute capability be >=70?

Reply:
According to the CUDA documentation, Tensor Core requires 70 at least.
https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities